content/applications/inventory_and_mrp/manufacturing/workflows/allocation.rst

```diff
@@ -0,0 +1,104 @@
+manufactured, and whether or not there are any |SOs| or |MOs| which require the product, but have
+not already been allocated units of it.
+
+If there **ARE** existing |SOs| and |MOs| that require the product, **AND** there are too few units
+of the product on-hand to fulfill those orders, then a :icon:`fa-list` :guilabel:`Allocation` smart
+button appears at the top of the page as soon as the |MO| is confirmed.
+
+If there **ARE** existing |SOs| and |MOs| that require the product, **AND** there **ARE** enough
+units of the product on-hand to fulfill those orders, then the :icon:`fa-list`
+:guilabel:`Allocation` smart button only appears at the top of the page once the |MO| has been
+marked as done, by clicking :guilabel:`Produce All`.
+
+.. image:: allocation/allocation-button.png
+   :align: center
+   :alt: The Allocation smart button at the top of an MO.
+
+.. note::
+   If there **ARE NOT** any existing |SOs| and |MOs| that require the product, the :icon:`fa-list`
+   :guilabel:`Allocation` smart button does not appear, even when the |MO| has been marked as done.
+
+Click the :icon:`fa-list` :guilabel:`Allocation` smart button to open the *MRP Reception Report* for
+the |MO|.
+
+If a finished product is manufactured, this page lists any open customer delivery orders for which
+quantities of the product have yet to be reserved.
+
+If a manufacturing component is manufactured, this page lists any open |MOs| for which quantities of
+the component have yet to be reserved.
+
+Click the :guilabel:`Assign All` button to the right of a specific order to assign products for each
+quantity required to fulfill that order. For example, if an order requires one quantity of four
+units of the product, and one quantity of one unit of the product, clicking :guilabel:`Assign All`
+assigns five units of the product to fulfill both quantities.
+
+Alternatively, click :guilabel:`Assign` next to a specific quantity to only assign products to that
+quantity, and not any others in the order. Using the previous example, clicking :guilabel:`Assign`
+next to the quantity of one unit assigns a product to that quantity, but leaves the quantity of four
+units without any products assigned.
+
+After assigning products to a quantity, the :guilabel:`Assign` button turns into an
+:guilabel:`Unassign` button. Click :guilabel:`Unassign` to unreserve the assigned products from that
+quantity, making them available for other quantities.
+
+.. image:: allocation/reception-report.png
+   :align: center
+   :alt: The MRP Reception Report for an MO.
+
+After clicking :guilabel:`Assign All` or :guilabel:`Assign`, the :guilabel:`Print Labels` or
+:guilabel:`Print Label` button to the right of it becomes selectable. Selecting either button
+generates and downloads a PDF document with one label for each product that was assigned. These
+labels are used to designate each product as being reserved for that specific order.
+
+.. image:: allocation/assigned-labels.png
```

Comment:
This image is confusing for me - I couldn't get my runbots to do this flow, so I couldn't reproduce this image. It's repeating all the same info on each slip- is there a larger pic with more info that you can add here?

Reply:
Each label should have the same info since all of the products have been allocated to the same order.